### DISCUSSION TITLE
[4.0] UX - Copy Item missing from article actions dropdown

### DISCUSSION THREAD

**Discussion by coolcat-creations** [joomla/joomla-cms#28864] [2. Cold Features]
### Is your feature request related to a problem? Please describe.
Often you want just to copy an article - like you would copy a module, but it's not possible unless you use the batch process modal.
It would be great to have a copy action in action dropdown like in modules. That would be also more consistent through the system.

<img width="456" alt="grafik" src="https://user-images.githubusercontent.com/828371/80570333-dd8c2500-89fa-11ea-961c-71f42d4ff2e1.png">

**Comment by brianteeman** [comment]
Do we really have to have multiple options for everything?

**Comment by coolcat-creations** [comment]
For Copy yes, it's a very often used task and it's also available in modules.
To be more consistent in the UI I would strongly recommend to add copy here.
It's 3 clicks to copy an article against 6 clicks for the user.

**Comment by coolcat-creations** [comment]
Sidenote: If the article is not featured / not checked out I see no reason to show an active unfeature / checkin action there

**Comment by brianteeman** [comment]
> For Copy yes, it's a very often used task and it's also available in modules.

According to what metric?
Personally I only ever use Save as copy

> To be more consistent in the UI I would strongly recommend to add copy here.

Then you would have to add it in a lot more places than just articles

**Comment by simbus82** [comment]
> > For Copy yes, it's a very often used task and it's also available in modules.
> 
> According to what metric?
> Personally I only ever use Save as copy

According to experienced admin users like me 🥇 for example. :-P

Me, my copywriters and my devs always use the "copy" for K2 articles in J3, because many times it's needed to create a "template" for an article. Populating a web site prototype is so easy with an article duplication system. I missing this for standard articles in J3, it's so annoying enter and click save as copy.
And in this manner i can't duplicate 2 or more different titled articles in one shot.

> Then you would have to add it in a lot more places than just articles

Exactly. A rapid "copy" can be a must for menu voices too. No need other options because a simply copy button must clone the menu voice as-is at the same level.
For all others mass management cases, we have the batch.

**Comment by alikon** [comment]
please test #28881

**Comment by alikon** [comment]
as #28881 it is still a WIP..... let's reopen this maybe someonelse will be more fast than me :smiley:
